raiden-dapp/vue.config.js

```diff
@@ -5,12 +5,9 @@ module.exports = {
   productionSourceMap: false,
   // https://forum.vuejs.org/t/solution-to-building-error-in-circleci-or-any-other-machine-with-cpu-limitations/40862
   parallel: !process.env.CIRCLECI,
-  publicPath:
-    process.env.NODE_ENV !== 'production'
-      ? './'
-      : process.env.DEPLOYMENT === 'staging'
-      ? '/staging/'
-      : '/',
+  publicPath: process.env.VUE_APP_PUBLIC_PATH
+    ? process.env.VUE_APP_PUBLIC_PATH
+    : '/',
```

Comment:
We need to fallback to relative paths (`./`) if on development mode, or else this breaks the serve_pr script

Reply:
Ah thats what it is for. Wait a seconds I'll find a solution.

Reply:
Here we go. I'll test it using the serve pr script as soon as the latest CI run through.